rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala

```diff
@@ -3,14 +3,15 @@ package coop.rchain.rspace
 import java.nio.ByteBuffer
 import java.nio.file.Path
 
-import org.lmdbjava.{Dbi, Env, Txn}
+import org.lmdbjava.{Dbi, Env, Txn, TxnOps}
 import scodec.Codec
 import coop.rchain.shared.ByteVectorOps._
 import coop.rchain.shared.AttemptOps._
 import coop.rchain.shared.PathOps._
 import scodec.bits.BitVector
-
 import kamon._
+import org.lmdbjava.Library.LIB
```

Comment:
[error] /home/travis/build/rchain/rchain/rspace/src/main/scala/coop/rchain/rspace/LMDBOps.scala:13:21: object Library in package lmdbjava cannot be accessed in package org.lmdbjava